index.html

```diff
@@ -9265,6 +9265,38 @@ <h3>Graph Containers</h3>
     <a>processing mode</a> set to <code>json-ld-1.1</code>.</p>
 </section>
 </section>
+
+<section class="informative"><h3>Loading Documents</h3>
+  <p>The JSON-LD 1.1 Processing Algorithms and API specification [[JSON-LD11-API]]
+    defines the interface to a <a>JSON-LD Processor</a> and includes
+    a number of methods used for manipulating different forms
+    of JSONLD (see <a href="#forms-of-json-ld" class="sectionRef"></a>).
+    This includes a general mechanism for loading remote documents,
+    including referenced JSON-LD documents and remote contexts,
+    and extracting embedded JSON-LD from other formats such as [[HTML]].
+    This is more fully described in
+    <a data-cite="JSON-LD11-API#remote-document-and-context-retrieval">Remote Document and Context Retrieval</a>
+    in [[JSON-LD11-API]].</p>
+
+  <p>A <dfn data-cite="JSON-LD11-API#dom-jsonldoptions-documentloader">documentLoader</dfn>
+    can be useful in a number of contexts where loading remote documents can be problematic:</p>
+  <ul>
+    <li>Remote context documents should be cached to prevent overloading the
+      location of the remote context for each request.
+      Normally, an HTTP caching infrastructure might be expected to handle this,
+      but in some contexts this might not be feasible.
+      A <a>documentLoader</a> implementation might provide separate logic for performing
+      such caching.</li>
+    <li>Non-standard URL schemes may not be widely implemented,
+      or may have behavior specific to a given application domain.
+      A <a>documentLoader</a> can be defined to implement document retrieval semantics.</li>
+    <li>Certain well-known contexts may be statically cached within a <a>documentLoader</a> implementation.
+      This might be particularly useful in embedded applications,
+      where it is not feasible, or even possible, to access remote documents.</li>
+    <li>For security purposes, the act of remotely retriving a document may provide a signal of application behavior.
+      The judicious use of a <a>documentLoader</a> can isolate the application and reduce its online footprint.</li>
```

Comment:
shouldn't that be
```suggestion
      The judicious use of a <a>documentLoader</a> can isolate the application and reduce its online fingerprint.</li>
```
?
For me, "footprint" would be more about bandwidth, CPU load, and caching (as in the first item), and "finderprint" about security... But I might be wrong.